app/build.gradle

```diff
@@ -37,6 +37,7 @@ android {
             useJUnitPlatform()
         }
         animationsDisabled = true
+        unitTests.returnDefaultValues = true
```

Comment:
```suggestion
```
remove this as this changes the behaviour of a lot of tests

Reply:
Nvm i recalled i time where i had this and it borked an existing test but now its not happening